ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -209,12 +207,10 @@ export const CoinOverview = ({
               className: 'wallet-overview__portfolio_button',
             }}
             size={ButtonLinkSize.Inherit}
-            textProps={{
-              variant: TextVariant.bodyMdMedium,
-            }}
             onClick={handlePortfolioOnClick}
             as="a"
             data-testid="portfolio-link"
+            className="wallet-overview__portfolio_text"
           >
```

Comment:
`ButtonLink` component was not applying variant: `TextVariant.bodyMdMedium` i sent in textProps;
Im passing className instead to overrida text css changes;

Reply:
Why not this?
```
<ButtonLink
endIconName={IconName.Export}
onClick={handlePortfolioOnClick}
as="a"
data-testid="portfolio-link"
className="wallet-overview__portfolio_text"
>
{t('portfolio')}
</ButtonLink>
```


https://github.com/user-attachments/assets/2e5390ed-2131-4da4-a226-0aaad4c6c6d7

Reply:
This works too! 🙏  just adding marginLeft to keep '8px' between portfolio text and portfolio export icon
```
          <ButtonLink
            endIconName={IconName.Export}
            endIconProps={{
              marginLeft: 2,
            }}
            onClick={handlePortfolioOnClick}
            as="a"
            data-testid="portfolio-link"
            className="wallet-overview__portfolio_text"
          >
            {t('portfolio')}
          </ButtonLink>
```
but i see in the suggestion you are using `className="wallet-overview__portfolio_text"`,
Initially i wanted to use textProps to pass my props; but they were not applied
 textProps={{ variant: TextVariant.bodyMdMedium }}

![image](https://github.com/user-attachments/assets/d8469878-e3dd-4be9-b9ce-b366c04db2af)


One other solution would be to not use ButtonLink component and do something that looks like this with few tweaks
```
      <Box
            onClick={handlePortfolioOnClick}
            data-testid="portfolio-link"
          >
            <Text variant={TextVariant.bodyMd} color={TextColor.primaryDefault}>
              {t('portfolio')}
            </Text>

            <Icon
              size={IconSize.Sm}
              name={IconName.Export}
              color={IconColor.primaryDefault}
            />
          </Box>
```

Reply:
Resolving this; this is fixed here [52a8669](https://github.com/MetaMask/metamask-extension/pull/27161/commits/52a8669072e05aef09347df60e9ae1459dd543d3) 🙏

Reply:
It seems like there's a bug or it's not very intuitive. When `ButtonLink` uses `size="inherit"` or `size="auto"` (e.g. `<ButtonLink size={ButtonLinkSize.Inherit}>`, `<ButtonLink size={ButtonLinkSize.Auto}>`), the CSS applies `inherit` to the font styles, which prevents the style utility props from working. cc @garrettbear